src/coreComponents/math/UOM.hpp

```diff
@@ -0,0 +1,49 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 TotalEnergies
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file UOM.hpp
+ * @brief Regroups useful constants and functions relative to Units Of Measures
+ * that are globally used for math and physics computations.
+ */
+#ifndef GEOS_MATH_UOM_HPP_
+#define GEOS_MATH_UOM_HPP_
```

Comment:
```suggestion
 * @file PhysicsConstants.hpp
 * @brief Regroups useful physics constants that are globally used for math and physics computations in solvers
 */
#ifndef GEOS_MATH_PHYSICSCONSTANTS_HPP_
#define GEOS_MATH_PHYSICSCONSTANTS_HPP_
```
Three other comments:
- `Constants` or `PhysicsConstants` are also good naming options. Maybe ultimately better than units if we want to add gravitational acceleration, etc
- You should probably advertise the addition of this file on Slack to make sure that people add their constants to it
- You could consider adding this file to `common` instead of `math` because this file is not exactly related to math operations

Reply:
OK, I changed `UOM.hpp` to `PhysicsConstants.hpp` and moved it to `common/`.